constellations/orion/src/repos/migrations/000026_create_table_user_afh.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE user_afh
+(
+    id          int unsigned NOT NULL AUTO_INCREMENT,
+    user_id     int          NOT NULL,
+    afh_id      int          NOT NULL,
```

Comment:
unsigned int. Should match the column type of what you see in the ask for help table.